crypto/ec/ecx_meth.c

```diff
@@ -1061,18 +1098,18 @@ static int s390x_pkey_ecx_keygen25519(EVP_PKEY_CTX *ctx, EVP_PKEY *pkey)
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
     };
-    ECX_KEY *key;
+    ECX_KEY *key = NULL;
```

Comment:
Setting this to NULL seems unnecessary given it is unconditionally assigned three lines later.

Reply:
Fixed - and the other similar occurrences.